src/main/java/com/roommatematcher/enterprise/RoomMateController.java

```diff
@@ -26,4 +35,17 @@ public String index(Model model){
         model.addAttribute(profile);
         return "start";
     }
+
+    @PostMapping(value="/profile", consumes="application/json", produces="application/json")
+    public Profile createProfile(@RequestBody Profile profile) throws Exception {
+
+        Profile newProfile = null;
+        try{
+            newProfile = profileService.save(profile);
+        } catch (Exception e) {
+            //TODO ADD LOGGING
+            System.out.println("tried to access methods in business logic but failed");
+        }
+        return newProfile;
```

Comment:
Careful here.  Since you're not handling the exception, and the variable is initialized to null, it's possible that this variable could hold, and thus return, a null.  That's potentially worse than the exception that is getting caught and discarded.  Recommendation: Take a careful look at exception best practices, and consider how to use them here.